rosidl_generator_cpp/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ project(rosidl_generator_cpp)
 
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
+find_package(rosidl_generator_c REQUIRED)
+include_directories(${rosidl_generator_c_INCLUDE_DIRS})
```

Comment:
Should these two lines be moved into the `BUILD_TESTING` block (and the `include_directories` call be merged with the existing call there)?

Reply:
Done.